src/components/routes/Trips/StartedTripCard.container.ts

```diff
@@ -0,0 +1,187 @@
+import styled from 'styled-components';
+import { color, typography } from 'styled/utils';
+
+const StartedTripCardMobile = styled.div`
+  box-shadow: 0 4px 15px ${color('black', 0.1)};
+  display: flex;
+  flex-direction: column;
+  min-width: 272px;
+  .started-trip-info {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    padding: 24px;
+    width: 100%;
+    h3 {
+      ${typography('title', 7)};
+      color: ${color('body')};
+      margin-bottom: 8px;
+    }
+    .address {
+      margin-bottom: 16px;
+      .bee-list-item {
+        span {
+          ${typography('emp', 7)}
+        }
+      }
+    }
+    .dates {
```

Comment:
any reason to redefine dates css?
shouldn't this be the same css as other date css in cards?

Reply:
currently we are defining the styles explicitly for each `StartedTripCard`, `ActiveTripCard`, and `ExpiredTripCard`. Should we make a reusable "CardInfo" component that has the styles?

Reply:
yeah, i think they are the same styles?

Reply:
the sketch file has minor variations such as the dates being placed in different locations, the google maps icon exists for Active Cards, and different fabs depending on the state (this is already being done in our existing cards with condition rendering, but will be more logic in one reusable component). Should we consolidate into one component and forgo the small design differences? If so, I'm thinking I may need a props flag for cardSize: `big` and `small`.

![image](https://user-images.githubusercontent.com/18321302/51702658-359ed500-1fc9-11e9-976b-e95d3e0e624d.png)

Reply:
i see, the difference is the black "checkin date" and the  "12/01/2017" in the small card.
i think its fine to keep the difference then.